openchain-blockchain-coinprism/azuredeploy.parameters.json

```diff
@@ -9,7 +9,7 @@
             "value": "Standard_A0"
         },
         "openchainVersion": {
-            "value": "0.5.0"
+            "value": "0.6.2"
```

Comment:
Just remove this from the param file since it defaults to the only allowed version